css/css-flexbox/flexbox_mix-writing-flex-alignment.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <title>Flexbox | Flex alignment and writing modes</title>
+  <meta charset="utf-8">
+  <meta name="assert" content="Flex alignment and writing modes">
+  <link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#pagination">
+  <link rel="author" title="Clayton Salem" href="claytonjsalem@gmail.com">
+  <style>
+    /* Writing mode utility classes. */
+    .horizontal-tb { writing-mode: horizontal-tb; }
+    .vertical-rl { writing-mode: vertical-rl; }
+    .vertical-lr { writing-mode: vertical-lr; }
+    .sideways-rl { writing-mode: sideways-rl; }
+    .sideways-lr { writing-mode: sideways-lr; }
+    /* Positioning utility classes. */
+    .start { justify-content: start; }
+    .flex-start { justify-content: flex-start; }
+    .flex-end { justify-content: flex-end; }
+    .space-between { justify-content: space-between; }
+    .space-around { justify-content: space-around; }
+    .space-evenly { justify-content: space-evenly; }
+    .flex-end { align-items: flex-end; }
+    .flex-start { align-items: flex-start; }
+    .baseline { align-items: baseline; }
+    .stretch { align-items: stretch; }
+    .center { align-items: center; }
+    .row { flex-direction: row; }
+    .row-reverse { flex-direction: row-reverse; }
+    .column { flex-direction: column; }
```

Comment:
I don't see these ones being applied in the JS below, have you not done that logic yet or am I missing it?

Reply:
Good catch, thanks.